src/EurekaClient.js

```diff
@@ -303,19 +303,21 @@ export default class Eureka {
    */
   transformRegistry(registry) {
     if (!registry) {
-      throw new Error('Unable to transform empty registry');
-    }
-    this.cache = { app: {}, vip: {} };
```

Comment:
I'm good with that theory. I still like the changes as is, so since its functionally the same I'm going to keep them. Not to mention the tests are a little better too. Any objections?